docs/Using-Virtual-Environment.md

```diff
@@ -0,0 +1,33 @@
+# Using Virtual Environment
+
+## What is a Virtual Environment?
+A Virtual Environment is a walled garden for Python projects.  
+
+## Why should I use a Virtual Environment?
+A Virtual Environment keeps all dependencies for the project separate from dependencies 
+of other projects. This has a few advantages:
+1. Firstly, it makes dependency management for the project easy.
```

Comment:
remove Firstly.  Might also add "it allows you to separate and isolate the dependencies between projects"

Reply:
Done